lib/DDG/Spice/Metar.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::Metar;
+
+# Metar: Looks up the METAR information for a given airport. 
+# The input should be a 4 letter ICAO code.
+# It recognizes the query "metar ICAO" and "ICAO metar". 
+
+use DDG::Spice;
+
+spice to => 'http://avwx.rest/api/metar.php?station=$1&format=JSON';
+spice proxy_cache_valid => "418 1d"; # Do not cache responses.
+spice wrap_jsonp_callback => 1;
+
+triggers any => 'metar';
```

Comment:
Let's use `startend` because we're only expecting `metar` to be the first or last word in the query